src/Routing/Router.php

```diff
@@ -987,8 +990,14 @@ public static function prefix($name, $params = [], $callback = null)
             $callback = $params;
             $params = [];
         }
-        $name = Inflector::underscore($name);
-        $path = '/' . $name;
+
+        if (empty($params['path'])) {
+            $path = '/' . Inflector::underscore($name);
```

Comment:
This causes a slight change in behavior. Earlier the `$name` var was also underscored but it isn't now. This can cause problems to someone if he has used `Router::prefix('Admin', ..)` instead of `Router::prefix('admin', ...)`.  So when `$params['path']` is not set `$name` should also be underscored.